app/views/schools/discord_server_roles.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= render "discord_header", locals: { tab: :discord_server_roles } do %>
+  <% if params[:cause].present? && params[:cause] == "DeletedRoles" && @sync_service.sync_ready? %>
+    <%= render "discord_roles_diff", locals: { sync_service: @sync_service } %>
```

Comment:
It isn't mandatory to pass local variables using the `locals:` hash. You can simply pass the required variables directly:

```suggestion
    <%= render "discord_roles_diff", sync_service: @sync_service %>
```

Not a necessary change - saying `locals: { ... }` is a common format.